packages/components/e2e/component-objects/src/main/java/org/talend/component/List.java

```diff
@@ -74,4 +108,19 @@ public Table getTable() {
     public Table getTable(final String id) {
         return new Table(driver, id);
     }
+
+    /**
+     * Get the table display element manager.
+     *
+     * @return table the table element manager
+     */
+    public Display getDisplay() {
+        Large large = new Large(driver);
+        try {
+            large.getElementToScroll();
```

Comment:
Why do you get the scroll element ?
Use `getElement()` instead please. Or use display selectors, I think there is a `tc-list-large` or classes like that